dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -755,3 +755,50 @@ CWWKD1077.defaultds.not.found.explanation=Jakarta Data repositories default to \
  configuration.
 CWWKD1077.defaultds.not.found.useraction=Configure the Jakarta EE default \
  data source or specify a different data store for the repository.
+
+CWWKD1078.datastore.not.found=CWWKD1078E: The {0} application artifact has a \
+ {1} repository interface with a {2} data store that does not match the Id of a \
+ dataSource element or a databaseStore element in server configuration, or it does \
+ not match the JNDI name of a data source, a data source resource reference, or \
+ a persistence unit reference that is accessible to the application artifact, \
+ or one of the related resources is not configured correctly. \
+ A dataSource element can be defined in server configuration. For example, {3} \
+ A databaseStore element can be defined in server configuration. For example, {4} \
+ An application component can use the Resource annotation to define a \
+ data source resource reference. For example, {5}, which requires server \
+ configuration to have a dataSource configuration element with a {6} attribute. \
+ An application component can use the PersistenceUnit annotation to define a \
+ persistence unit reference. For example, {7}, which requires the persistence.xml \
+ file to have a persistence-unit element with a matching name attribute. \
+ For example, {8}
```

Comment:
Similar to CWWKD1077 (from a previous PR), the `.` punctuation is omitted after certain examples where it would cause the example to have an awkward appearance in the message.  This applies to the CWWKD1079 message as well.